### PR TITLE
🐛 Disable selection in IssueCommentHtmlCell

### LINKED ISSUE #608: Make HTML template a Swift 4 String
We can do multiline strings like python now, right? [This is gross](https://github.com/rnystrom/GitHawk/blob/master/Classes/Issues/Comments/Html/IssueCommentHtmlCell.swift#L25-L53).

### PR DESCRIPTION
**Pull request by viktorgardart**
Now you can select and scroll the "Sent with Githawk" footer text. 

I only tried this with the Signature and "Bug report dump" not sure where else we're using the `IssueCommentHtmlCell`. 

Also Closes #608

### CODE REVIEW

**Comment by rnystrom** [comment]
Is this bad tho?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by viktorgardart** [comment]
@rnystrom I thought about it before creating the PR. 
It sure isn't bad, but feels weird to only be able to select the HTML parts of a comment. I would say either disable it in the HTML or allow it on native text as well! 🙂

File: Classes/Issues/Comments/Html/IssueCommentHtmlCell.swift [hunk 1]
```diff
@@ -123,6 +123,8 @@ final class IssueCommentHtmlCell: UICollectionViewCell, ListBindable, UIWebViewD
     }
 
     func webViewDidFinishLoad(_ webView: UIWebView) {
+        webView.stringByEvaluatingJavaScript(from: "document.documentElement.style.webkitUserSelect='none'")
+        webView.stringByEvaluatingJavaScript(from: "document.documentElement.style.webkitTouchCallout='none'")
```
Comment (rnystrom) [inline Classes/Issues/Comments/Html/IssueCommentHtmlCell.swift]: Shouldn't this be added into the HTML template?

https://github.com/rnystrom/GitHawk/blob/master/Classes/Issues/Comments/Html/IssueCommentHtmlCell.swift#L25-L53

**Comment by viktorgardart** [comment]
@rnystrom Made the template into a multiline string and moved the javascript to the bottom of the body 👌

**Comment by rnystrom** [comment]
@viktorgardart heads up that this broke some styles. `<detail>`s look like they are Times New Roman or something now.

**Comment by viktorgardart** [comment]
@rnystrom that’s odd, I might have time to look into it on the plane later today! Otherwise tomorrow! 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**